Src/VimCore/Modes_Insert_InsertMode.fs

```diff
@@ -865,6 +876,27 @@ type internal InsertMode
         | Some insertCommand ->
             match insertCommand.RightMostCommand with
             | InsertCommand.Insert text -> func text
+            | InsertCommand.Back ->
+                if _globalSettings.Digraph && keyInput.RawChar.IsSome then
+                    match insertCommand.SecondRightMostCommand with
+                    | Some (InsertCommand.Insert text) when text.Length > 0 -> 
+
+                        // The user entered 'char1 <BS> char2' and digraphs are
+                        // enabled, so check whether 'char1 char2' is a digraph
+                        // and if so, insert it.
+                        let firstKeyInput =
+                            text.[text.Length - 1]
+                            |> KeyInputUtil.CharToKeyInput
+                        let secondKeyInput = keyInput
+                        match x.TryInsertDigraph firstKeyInput secondKeyInput with
+                        | Some processResult ->
+                            Some processResult
+                        | None ->
+                            x.TryInsertDigraph secondKeyInput firstKeyInput

```

Comment:
Good catch. Didn't see any documentation about it trying the reverse diagraph here.

Reply:
Yes, it's in there!

> If a digraph with {char1}{char2} does not exist, Vim searches for a digraph
> {char2}{char1}.  This helps when you don't remember which character comes
> first.